crates/holochain_types/src/dna/dna_file.rs

```diff
@@ -21,8 +20,8 @@ mod test;
     derive_more::AsRef,
     derive_more::From,
     derive_more::IntoIterator,
-    Arbitrary,
 )]
+#[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary))]
```

Comment:
Is this part of this PR?

Reply:
Not sure, might have been a quick addition